streampark-console/streampark-console-service/src/main/assembly/script/upgrade/mysql/2.1.0.sql

```diff
@@ -88,52 +42,192 @@ create table `t_yarn_queue` (
+update `t_role_menu` set menu_id=110502 where menu_id=100045;
+update `t_role_menu` set menu_id=110503 where menu_id=100046;
+update `t_role_menu` set menu_id=110504 where menu_id=100062;
+update `t_role_menu` set menu_id=110600 where menu_id=100047;
+update `t_role_menu` set menu_id=110601 where menu_id=100048;
+update `t_role_menu` set menu_id=110602 where menu_id=100049;
+update `t_role_menu` set menu_id=110603 where menu_id=100050;
+update `t_role_menu` set menu_id=110604 where menu_id=100051;
+update `t_role_menu` set menu_id=110605 where menu_id=100063;
+update `t_role_menu` set menu_id=120000 where menu_id=100013;
+update `t_role_menu` set menu_id=120100 where menu_id=100014;
+update `t_role_menu` set menu_id=120101 where menu_id=100017;
+update `t_role_menu` set menu_id=120102 where menu_id=100022;
+update `t_role_menu` set menu_id=120103 where menu_id=100023;
+update `t_role_menu` set menu_id=120104 where menu_id=100035;
+update `t_role_menu` set menu_id=120105 where menu_id=100065;
+update `t_role_menu` set menu_id=120200 where menu_id=100015;
+update `t_role_menu` set menu_id=120201 where menu_id=100016;
+update `t_role_menu` set menu_id=120202 where menu_id=100018;
+update `t_role_menu` set menu_id=120203 where menu_id=100020;
+update `t_role_menu` set menu_id=120204 where menu_id=100021;
+update `t_role_menu` set menu_id=120205 where menu_id=100024;
+update `t_role_menu` set menu_id=120207 where menu_id=100026;
+update `t_role_menu` set menu_id=120208 where menu_id=100027;
+update `t_role_menu` set menu_id=120209 where menu_id=100028;
+update `t_role_menu` set menu_id=120210 where menu_id=100029;
+update `t_role_menu` set menu_id=120211 where menu_id=100030;
+update `t_role_menu` set menu_id=120212 where menu_id=100031;
+update `t_role_menu` set menu_id=120213 where menu_id=100032;
+update `t_role_menu` set menu_id=120214 where menu_id=100036;
+update `t_role_menu` set menu_id=120215 where menu_id=100042;
+update `t_role_menu` set menu_id=120216 where menu_id=100066;
+update `t_role_menu` set menu_id=120300 where menu_id=100053;
+update `t_role_menu` set menu_id=120301 where menu_id=100054;
+update `t_role_menu` set menu_id=120302 where menu_id=100055;
+update `t_role_menu` set menu_id=120303 where menu_id=100056;
+update `t_role_menu` set menu_id=120304 where menu_id=100057;
+update `t_role_menu` set menu_id=120305 where menu_id=100058;
+update `t_role_menu` set menu_id=120306 where menu_id=100067;
+update `t_role_menu` set menu_id=120307 where menu_id=100069;
+update `t_role_menu` set menu_id=130101 where menu_id=100068;
+update `t_role_menu` set menu_id=130102 where menu_id=100034;
+update `t_role_menu` set menu_id=120206 where menu_id=100025;
+update `t_role_menu` set menu_id=130000 where menu_id=100033;
+update `t_role_menu` set menu_id=130401 where menu_id=100040;
+update `t_role_menu` set menu_id=130402 where menu_id=100041;
+
+insert into `t_role_menu` (role_id, menu_id) values (100001, 120104);
```

Comment:
AnchorA(c).
(I don't checked it on the PC env. Because I saw the new commits till the current review.)

Will here(all tags bout AnchorA) exist redundant or conflict logic cases ?
- redundant: 
  - There would be no records about menu_id `120104` at `AnchorA(b)`, so the line(`AnchorA(c)`) could be executed.
- conflict:
  - If there are records about (100001, 100035) before executing `AnchorA(b)`, the line `AnchorA(c)`  would fail due to the duplicated unique-key constraint.     

That the case would be `redundant ` or `conflict` depends on the users' status of the production env. It's complex for us to cover.   
Bases on mentioned above,      
what is the bounds for the PR back-forward compatibility is impact for the expected state of the delta sql file(2.1.0.sql). 
In other words, should the scope of versions evolution we should do BF-compatibility be `release-2.0.0 -> now` or `release-2.0.0-> some users' cherrypick(into Prod env)  -> now` ?
What about defining the bounds before the next update ?

Looking forward to discussion & suggestions about it.
Thank you~.

Reply:
> AnchorA(c). (I don't checked it on the PC env. Because I saw the new commits till the current review.)
> 
> Will here(all tags bout AnchorA) exist redundant or conflict logic cases ?
> 
> * redundant:
>   
>   * There would be no records about menu_id `120104` at `AnchorA(b)`, so the line(`AnchorA(c)`) could be executed.
> * conflict:
>   
>   * If there are records about (100001, 100035) before executing `AnchorA(b)`, the line `AnchorA(c)`  would fail due to the duplicated unique-key constraint.
> 
> That the case would be `redundant ` or `conflict` depends on the users' status of the production env. It's complex for us to cover. Base on mentioned above, what is the bounds for the PR back-forward compatibility is impact for the expected state of the delta sql file(2.1.0.sql). In other words, should the scope of versions evolution we should do BF-compatibility be `release-2.0.0 -> now` or `release-2.0.0-> some users' cherrypick(into Prod env) -> now` ? What about defining the bounds before the next update ?
> 
> Looking forward to discussion & suggestions about it. Thank you~.

Thanks for your efforts for current review